homeassistant/components/netatmo/cover.py

```diff
@@ -42,6 +47,14 @@ def _create_entity(netatmo_device: NetatmoDevice) -> None:
         async_dispatcher_connect(hass, NETATMO_CREATE_COVER, _create_entity)
     )
 
+    platform = entity_platform.async_get_current_platform()
+
+    platform.async_register_entity_service(
+        SERVICE_MOVE_COVER_TO_PREFERRED_POSITION,
+        None,
+        "_async_service_move_cover_to_preferred_position",
+    )
```

Comment:
As this is a service without input, I think this should be a button entity and not a service.

../Frenck

Reply:
The main reason i choose to make an service is that not all roller shutter have the functionnality and with the API there is no solution to detect it. 
With a button, all netatmo user will see it, but it will not work. 
Maybe disabled by default ? what do you think is the best solution ?

Reply:
Changed to a button